### PR TITLE
Add endpoint for batch delete

### PR DESCRIPTION
**Pull request by allysonjp715**

### CODE REVIEW

File: data-serving/data-service/api/openapi.yaml [hunk 1]
```diff
@@ -136,6 +136,31 @@ paths:
           $ref: "#/components/responses/422"
         "500":
           $ref: "#/components/responses/500"
+    delete:
+      summary: Deletes multiple cases
```
Comment (attwad) [inline data-serving/data-service/api/openapi.yaml:140]: add tags: [Case] here please

File: verification/curator-service/api/openapi/openapi.yaml [hunk 2]
```diff
@@ -413,6 +413,31 @@ paths:
           $ref: "#/components/responses/422"
         "500":
           $ref: "#/components/responses/500"
+    delete:
+      summary: Deletes multiple cases
```
Comment (attwad) [inline verification/curator-service/api/openapi/openapi.yaml:417]: ditto, tags: [Case]

**Review by attwad — APPROVED** [review]

File: data-serving/data-service/api/openapi.yaml (hunk 1 above)
Comment (allysonjp715) [reply data-serving/data-service/api/openapi.yaml:140]: Done

File: verification/curator-service/api/openapi/openapi.yaml (hunk 2 above)
Comment (allysonjp715) [reply verification/curator-service/api/openapi/openapi.yaml:417]: Done

**Pull request merged**